rollup.config.js

```diff
@@ -25,7 +26,14 @@ function umd(inputFile, outputFile) {
       node({
         module: true,
         only: ['tslib']
-      })
+      }),
+      babel({
+        exclude: 'node_modules/**',
+        plugins: [
+          'annotate-pure-calls',
+          'dev-expression',
```

Comment:
Why plugins here and presets in `babel.config.js`?  I think it best to just put the config in one place.  This is split.

Reply:
Thes are bundling plugins and not literal babel plugins